internal/slack/messages.go

```diff
@@ -42,10 +43,23 @@ func TransformSlack(inputFilePath, outputFilePath, team, attachmentsDir string)
 		return err
 	}
 
+	// TODO maybe change mmetl to include the correct paths during
+	// Transform -- however this seems to be fairly involved so for now
+	// just fix these paths after the fact
+	for _, post := range intermediate.Posts {
+		if len(post.Attachments) < 1 {
+			continue
+		}
```

Comment:
nit: I believe this check could be skipped as it is handled by `range`.

Reply:
oh yes that is quite right